### ISSUE TITLE
[plugin-*] Plugins not registering on RunKit

### ISSUE THREAD

**Issue by larsgw** [citation-js/citation-js#2]
The same thing *is* working locally.

https://runkit.com/larsgw/citation-js-plugins-not-registering

```js
require("@citation-js/plugin-bibtex")
c = require("@citation-js/core")

c.plugins.list()
c.plugins.input.list()
```

**Comment by larsgw** [comment]
So RunKit doesn't hoist dependencies.

https://discuss.runkit.com/t/need-to-manually-require-peer-dep/117/4

**Comment by larsgw** [comment]
Fixed in 830c381261adaea9249b9ac83471136fc7b39a76
